feature-wallet-impl/src/main/java/jp/co/soramitsu/feature_wallet_impl/presentation/transaction/history/mixin/TransactionStateMachine.kt

```diff
@@ -0,0 +1,162 @@
+package jp.co.soramitsu.feature_wallet_impl.presentation.transaction.history.mixin
+
+import jp.co.soramitsu.common.data.model.CursorPage
+import jp.co.soramitsu.feature_wallet_api.domain.interfaces.TransactionFilter
+import jp.co.soramitsu.feature_wallet_api.domain.interfaces.allFiltersIncluded
+import jp.co.soramitsu.feature_wallet_api.domain.model.Operation
+
+object TransactionStateMachine {
+
+    const val PAGE_SIZE = 100
+    private const val SCROLL_OFFSET = PAGE_SIZE / 2
```

Comment:
why not keep it 100?

Reply:
if pagesize == scrollOffset, then the second page will be triggered to load immediately

Reply:
Got it, looked in the code once again, yeah, 50 is great to have. Thank you